CMakeLists.txt

```diff
@@ -206,14 +206,6 @@ set(Boost_USE_STATIC_LIBS   OFF)
 set(Boost_USE_MULTITHREADED ON)
 set(Boost_REALPATH ON)
 
-# Select python3 by default unless its specified in the configure file
-# message("Using component: python${BOOST_PYTHON_SUFFIX}")
-# if(BOOST_PYTHON_SUFFIX)
-#   FIND_PACKAGE(Boost COMPONENTS "python${BOOST_PYTHON_SUFFIX}" REQUIRED)
-# else(BOOST_PYTHON_SUFFIX)
-#   FIND_PACKAGE(Boost COMPONENTS python REQUIRED)
-# endif(BOOST_PYTHON_SUFFIX)
-
```

Comment:
@roigcarlo I know it was not you who commented out this, but note that without this finding boost is broken for everybody. Now we are forced to define `Boost_INCLUDE_DIRS` and `Boost_LIBRARY_DIRS` instead of any of the standard FindBoost inputs.

Reply:
Ok, fixing now

Reply:
My suggestion would be to:

```C++
find_package(Boost)

set(Boost_USE_STATIC_LIBS   OFF)
set(Boost_USE_MULTITHREADED ON)
set(Boost_REALPATH ON)
```

Since libs are no longer needed, pretty much any version of boost will suffice, included the system one

(that one seems to work in my computer)

Reply:
Thanks! I'll try it too.